packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+- Hover & onClick (Checkbox) Hover state & pressed state applies to checkbox component
+- MouseUp (Checkbox) Card and Checkbox are both in selected state, clicking again toggles components to unselected state
+
+### Context menu
+
+![Visual interaction of the context menu on cards using the mouse](./assets/context-interaction-mouse.png)
+
+A context menu can be triggered by right clicking the surface or through an action in the card.
+
+1.Rest
+
+2. RightClick [LongPress]
+   Right click anywhere on card surface invokes content menu
+3. Show
+   Context menu appears close to cursor location on card
+4. Hide/Dismiss
+   Clicking or tapping anywhere outside of menu to dismiss
+
+#### Disabled
+
+![Visual interaction of disabled cards using the mouse](./assets/disabled.png)
+
+1 Unselected disabled
+
+1. Label: Selectable Card
+2. Role: Group
+3. State: Disabled, unselected
+
+2 Selected disabled
+
+1. Label: Selectable Card
+2. Role: Group
+3. State: Disabled, selected
+
+### Keyboard
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the keyboard](./assets/selectable-interaction-keyboard.png)
+
+1. Rest Default state is ‘unselected’
+2. Focus Focus rectangle is shown on card
+3. Focus Use spacebar to toggle state (selected/unselected)
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the keyboard](./assets/non-interactive-interaction-keyboard.png)
+
+Keyboard interaction
+
+1. Focus Initial focus is set on card, to access items within the card use EnterKey or Spacebar
+2. Focus-Inside Focus moves to first actionable item in the card. Use ArrowKeys or TabKey to move focus to child elements within the card. To exit card use EscKey, to move focus back to card [parent]
+3. Checkbox-Focus Depending on your tab order of items within the card, the number of keystrokes will vary. Once focus is set on the checkbox by default it will be set to unselected
+4. SpaceBar or EnterKey will toggle both Card and Checkbox to selected state. Keying again will toggle to unselected
+
+### Context menu
+
+![Visual interaction of the context menu on cards using the keyboard](./assets/context-interaction-keyboard.png)
+
+1. Rest
+2. OnFocus - Focus rect. is set on card. Use shiftKey and press F10 to invoke context menu
+
+   `aria-haspopup="menu"`
+
+   `aria-expanded="false"`
+
+3. ShowSet focus on context menu
+
+   `aria-expanded="true"`
+
+4. Hide/Dismiss - Use EscKey to hide/dismiss
+
+   `aria-expanded="false"`
+
+## Accessibility
+
+> ⚠️ Note: All other accessibility information, not covered in this section, is provided throughout the spec.
+
+Each card is of `role="group"`.
+
+### Narrator guidance
+
+#### Selectable cards
+
+![Visual interaction of keyboard usage with selectable cards](./assets/selectable-narrator.png)
+
+1 Focus + unselected
+
+1. Label: Select this card
+2. Role: Group
+3. State: Focus + unselected
+4. Helper text: Use enterKey or spacebar to toggle state to selected
```

Comment:
there is no good way to add such custom helper text to screen reader narration. You can consider:
- aria-describedby linking to an element with the description text
- aria live narration when (first) card gets focused.
Both have pros and cons, I would suggest to set up a meeting with @kolaps33 and @smhigley to discuss pros and cons or alternatives.

Reply:
I suggest the discussion remains here so it's documented what the pros and cons are, to keep our reasoning for why we choose option A or B.